addons/stock/wizard/stock_rules_report.py

```diff
@@ -33,6 +34,10 @@ def default_get(self, fields):
         if 'warehouse_ids' in fields:
             company = product_tmpl_id.company_id or self.env.company
             warehouse_id = self.env['stock.warehouse'].search(self.env['stock.warehouse']._check_company_domain(company), limit=1).id
+            if not warehouse_id:
+                raise UserError(
+                    _("No warehouse found for company %(company)s. You must create a warehouse to use this feature.", company=company.name)
+                )
```

Comment:
```suggestion
                raise UserError(_(
                    "No warehouse found for company %(company)s. You must create a warehouse to use this feature.",
                    company=company.name
        ))
```
Ensure that single line not exceeds 100 characters in length.